pages/classes/index.js

```diff
@@ -0,0 +1,69 @@
+import ClassInviteTable from '../../components/ClassInviteTable';
+import { PrismaClient } from '@prisma/client';
+import Head from 'next/head';
+import Navbar from '../../components/navbar';
+import Link from 'next/link';
+import Layout from '../../components/layout';
+
+const prisma = new PrismaClient();
+
+export async function getServerSideProps() {
+  // Here is where (if our env file is missing in our web folder) our code fails.
+  // There is no way to reach our DB link since we cannot access our env file that states where it is.
+  const classrooms = await prisma.Classroom.findMany();
```

Comment:
here is it possible to only fetch the classrooms that belong to the classroomTeacherId of the teacher that is signed in with next auth?

Reply:
We are fixing this in our other branch. (joinPage)